scripts/run_integration_test.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+# --- Configuration ---
+CLIENT_PACKAGE_NAME="client" # <-- Replace with your client package name
+CONTRACTS_OUTPUT_FILE=".deploy-output.json"
+ANVIL_RPC_URL="http://127.0.0.1:8545"
+
+# --- 1. Start Anvil in the background ---
+echo "Starting Anvil..."
+anvil > /dev/null & 
```

Comment:
nit: might be nice to log to file, so we can `cat` it in the `trap`.

Reply:
I will log them into `anvil_logs.txt` (also added that to gitignore)